assets/css/app.scss

```diff
@@ -1,5 +1,7 @@
 /* This file is for your main application css. */
-@import "./phoenix.css";
+// @import "./phoenix.css";
```

Comment:
Comment out `phoenix.css` which just loads Milligram
and instead import the TodoMVC CSS files.